### ISSUE TITLE
RRD support is broken

### ISSUE THREAD

**Issue by aburzyn** [graphite-project/docker-graphite-statsd#61]
Since graphite runs in /opt/graphite virtualenv, it is necessary to install rrdtool pyhton package in there.  In the current state (109aba0b2d9970be10d6534d55ce32d979f2a8e8) , the situation is the same as not having any rrd tool installed and Graphite ends up ignoring rrd files.

These lines in the dockerfile would suffice as far as I checked:
```
RUN apt-get update && apt-get install -y librrd-dev
RUN . /opt/graphite/bin/activate && pip install rrdtool
```

**Comment by deniszh** [comment]
Fixed in #62, v1.1.4-8 released

**Comment by aburzyn** [comment]
Ok, that was fast! Thanks!
